Mu2eKinKal/src/ANNStrawHitUpdater.cc

```diff
@@ -9,18 +9,15 @@ namespace mu2e {
   using KinKal::VEC3;
   ANNStrawHitUpdater::ANNStrawHitUpdater(ANNSHUConfig const& annshuconfig) {
     mva_  = new MVATools(std::get<0>(annshuconfig));
+    mva_->initMVA();
     mvacut_ = std::get<1>(annshuconfig);
     nulldoca_ = std::get<2>(annshuconfig);
     std::string freeze = std::get<3>(annshuconfig);
     freeze_ = WHSMask(freeze);
-    std::cout << "ANNStrawHitUpdater " << " anncut " << mvacut_ << " null doca " << nulldoca_ << " freezeing " << freeze_ << std::endl;
-    mva_->initMVA();
-    mva_->showMVA();
-    // outlier cuts: for now hardcode.  These should come from a central source FIXME
-    mintdrift_ = -2.0;
```

Comment:
Do these 4 values get set elsewhere now - I don't see where but I could easily have missed them.

Reply:
They are initialized on declaration, as you suggested.  A better solution is planned for an upcoming PR.